src/pages/feedback/index.tsx

```diff
@@ -0,0 +1,122 @@
+import BackLink from "components/BackLink"
+import Button from "components/Button"
+import ContactLink from "components/ContactLink"
+import { ErrorSummary } from "components/ErrorSummary"
+import Form from "components/Form"
+import Layout from "components/Layout"
+import Link from "components/Link"
+import Paragraph from "components/Paragraph"
+import SuccessBanner from "components/SuccessBanner"
+import config from "lib/config"
+import { withAuthentication, withCsrf, withMultipleServerSideProps } from "middleware"
+import { GetServerSidePropsContext, GetServerSidePropsResult } from "next"
+import Head from "next/head"
+import { ParsedUrlQuery } from "querystring"
+import React from "react"
+import AuthenticationServerSidePropsContext from "types/AuthenticationServerSidePropsContext"
+import CsrfServerSidePropsContext from "types/CsrfServerSidePropsContext"
+import { isError } from "types/Result"
+import User from "types/User"
+import postFeedback from "useCases/postFeedback"
+import createRedirectResponse from "utils/createRedirectResponse"
+import { isPost } from "utils/http"
+
+export const getServerSideProps = withMultipleServerSideProps(
+  withAuthentication,
+  withCsrf,
+  async (context: GetServerSidePropsContext<ParsedUrlQuery>): Promise<GetServerSidePropsResult<Props>> => {
+    const { req, formData, csrfToken, authentication, currentUser } = context as CsrfServerSidePropsContext &
+      AuthenticationServerSidePropsContext
+
+    if (!currentUser || !authentication) {
+      return createRedirectResponse("/login")
+    }
+
+    if (isPost(req)) {
+      const { feedback } = formData as { feedback: string }
+      const feedbackResult = await postFeedback(feedback, currentUser?.emailAddress ? currentUser?.emailAddress : "")
```

Comment:
Wouldn't it be better to pass in `currentUser?.emailAddress` to the `postFeedback` function, and the parameter optional?

Then inside the `postFeedback()` function, you could check to see if the email address is defined and handle it accordingly (perhaps saying something like "an unknown user has submitted feedback"